internal/server/http/imposter_test.go

```diff
@@ -22,6 +22,138 @@ func TestNewImposterFS(t *testing.T) {
 	})
 }
 
+func TestImposterFS_FindImposters(t *testing.T) {
+	// Set up
+	const expected = 7
+	ifs, err := NewImposterFS("test/testdata/imposters")
```

Comment:
Ideally, would be nice to also have a nested imposter, so we have a regression that future-proofs it.

Reply:
Let's resolve this, as soon we'll have https://github.com/friendsofgo/killgrave/pull/139 in place.